attach/mutt_attach.c

```diff
@@ -487,6 +489,26 @@ int mutt_view_attachment(FILE *fp, struct Body *a, enum ViewAttachMode mode,
 
     mutt_rfc3676_space_unstuff_attachment(a, mutt_buffer_string(tmpfile));
 
+    /* check for multipart/related and save attachments with a Content-ID */
+    struct Body *related_ancestor = NULL;
```

Comment:
This is only used in the `if` clause, so we can reduce the scope of the variable